tests/indexes/tests.py

```diff
@@ -98,3 +99,24 @@ def test_no_index_for_foreignkey(self):
             'CREATE INDEX `indexes_articletranslation_article_no_constraint_id_d6c0806b` '
             'ON `indexes_articletranslation` (`article_no_constraint_id`)'
         ])
+
+
+class DeprecateIndexTogetherTests(TestCase):
```

Comment:
SimpleTestCase should be fine. I would use a separate test file so it can be deleted when the deprecation ends.